tests/uptane_test.cc

```diff
@@ -153,9 +153,9 @@ TEST(uptane, sign) {
 }
 
 /*
- * \verify{\tst{153}} Check that aktualizr creates provisioning files if they
```

Comment:
Grrr. That's annoying (but presumably not your fault). What version of clang-format do you have? Our recommended version at present is 3.8.